apps/ui/src/apis/rides/join-ride-request.ts

```diff
@@ -0,0 +1,58 @@
+import { Message } from '@communecar/types';
```

Comment:
Consider changing the name of the file to something that indicates that a decision has been made on the request to join the trip